vowpalwabbit/fb_parser/src/parse_example_flatbuffer.cc

```diff
@@ -98,10 +117,7 @@ bool read_span_flatbuffer(
         has_more = false;
         break;
       default:
-        std::stringstream sstream;
-        sstream << "Error parsing examples: " << std::endl;
-        THROW(sstream.str());
-        return false;
+        RETURN_IF_FAIL(result);
     }
 
     has_more &= !temp_ex[0]->is_newline;
```

Comment:
A comment here would be useful for us mere mortals :)

Reply:
I added some comments throughout